server/src/GraphqlHTTPRequest.ts

```diff
@@ -1,10 +1,27 @@
 import { DocumentNode } from 'graphql';
 
 import { httpRequest } from './HTTPRequest';
+import { UPDATE_INVESTIGATION_METADATA } from './DBService/InvestigationMetadata';
 
 export const graphqlURL = '/coronai/graphql';
 export const baseUrl = 'http://localhost:';
 
-export const graphqlRequest = (query: DocumentNode, variables?: any) => (
+export const graphqlRequest = (query: DocumentNode, requestHeaders: any, variables?: any) => (
```

Comment:
why adding the headers?

Reply:
because the graphqlRequest needs the headers for the variables

Reply:
exports the variables from the headers